libs/vkd3d/vkd3d_private.h

```diff
@@ -3063,6 +3063,7 @@ struct vkd3d_physical_device_info
     VkPhysicalDeviceSeparateDepthStencilLayoutsFeaturesKHR separate_depth_stencil_layout_features;
     VkPhysicalDeviceShaderIntegerDotProductFeaturesKHR shader_integer_dot_product_features;
     VkPhysicalDeviceFragmentShaderBarycentricFeaturesNV barycentric_features_nv;
+    VkPhysicalDeviceFragmentShaderBarycentricFeaturesKHR barycentric_features_khr;
```

Comment:
These two structs are aliases of each other, so imo it would be better to just have the KHR one.

Reply:
Ah wait, the OpExtension string is still different.

Reply:
Yes, we need to check later if we enabled KHR feature, so it's more straight forward to do it like this.